libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/scripts/cx_built_in_test.lua

```diff
@@ -59,8 +59,45 @@ assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 1), 'could not add '
 --]]
 local PREARM_BYPASS = bind_add_param('PREARM_DIS', 1, 0)
 
+-- fetches the aircraft configuration from the config file and prints the model name and version
+local function get_aircraft_config(file_name)
+    local model = nil
+    local model_version = nil
+
+    -- Open file try and read header
```

Comment:
Let's add a `logger:log_file_content(file_name)` here. That will get the config file into the logs.

Reply:
Thanks for this suggestion Bob, We now see the AircraftConfiguration.xml file embedded within logs:
![image](https://github.com/user-attachments/assets/76b2728c-0eb0-4c4d-8903-a01f92c327c8)

Note : As discussed, filenames in logs are limited to 16 characters. Hence we see - "AircraftConfigur" in UAV Log Viewer.